data/data-pipeline/data_pipeline/etl/sources/tree_equity_score/etl.py

```diff
@@ -83,8 +96,42 @@ def transform(self) -> None:
             pd.concat(tes_state_dfs), crs=tes_state_dfs[0].crs
         )
 
+        # rename ID to Tract ID
+        self.df.rename(
+            # Block group ID delegated to attribute in superclass
+            columns={"geoid": ExtractTransformLoad.GEOID_FIELD_NAME},
+            inplace=True,
+        )
+
     def load(self) -> None:
-        logger.info("Saving Tree Equity Score GeoJSON")
+        logger.info("Saving Tree Equity Score CSV")
         # write nationwide csv
         self.CSV_PATH.mkdir(parents=True, exist_ok=True)
-        self.df.to_file(self.CSV_PATH / "tes_conus.geojson", driver="GeoJSON")
+        self.df = self.df[
+            [
```

Comment:
Looks good! If you want, you can follow the conventions by moving this list to `self.COLUMNS_TO_KEEP` defined on the class – see `data/data-pipeline/data_pipeline/etl/sources/ejscreen/etl.py` for an example. 

But that's not required, this is fine too.

Reply:
Looks like you decided not to change this -- that's fine!